lib/content-services/src/lib/dialogs/folder.dialog.scss

```diff
@@ -32,3 +32,14 @@
 .adf-dialog-action-button:disabled > span {
     color: var(--theme-text-color);
 }
+@media screen and (max-width: 380px){
+    .adf-font-size-input{
+        font-size: 6px;
+    }
+    .title{
+        margin: 0 0 0px;
```

Comment:
you can just style the `.mat-dialog-title` class inside this component, without introducing separate class

Reply:
noted and resolved